back-office/src/pages/UserScreen/UserScreen.jsx

```diff
@@ -0,0 +1,63 @@
+//@flow
+import React from 'react'
+
+import './UserScreen.css'
+import { User } from '../../user';
+import { UserLine, NewUserLine } from '../../components/user'
+
+class UserScreen extends React.Component {
+  
+  props: {
+    users: User[],
+    userEdited: User,
+    addUser: any,
+    updateUser: any,
+    updateCurrentUserProperty: any,
+    resetCurrentUser: any,
+  }
+
+  componentWillMount() {
+  }
+
+  editUserHandleChange = (name, value) => {
+    this.props.updateCurrentUserProperty(name, value)
+  }
+
+  editUserHandleSave = (user) => {
+    if (user.id) {
+      this.props.updateUser(user)
+    } else {
+      this.props.addUser(user)
+    } 
+  }
+
+  editUserHandleReset = () => {
+    this.props.resetCurrentUser()
+  }
+
```

Comment:
same here, you can move the handlers defintion to the container and have a functional component